app-emulation/libspectrum/libspectrum-1.4.4.ebuild

```diff
@@ -26,6 +26,7 @@ src_configure() {
 		$(use_with audiofile libaudiofile)
 		$(use_with bzip2)
 		$(use_with gcrypt libgcrypt)
+		$(use_enable static-libs static)
```

Comment:
Rather than adding a use flag, please just hard-disable it.

Reply:
Well, what is the reasoning behind disabling `static-libs` use flag?

Reply:
We don't provide static-libs USE unless there is a very good reason for it.

Reply:
Ok, I will disable static libs.

`eix --use static-libs` finds 1200 matches out of 18891 packages, so 6% at least.

Reply:
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0302

To my surprise nothing seems to be depending on `app-emulation/fuse[static-libs]` in tree. If something does, it can be enabled easily.